documentation/modules/auxiliary/admin/kerberos/get_ticket.md

```diff
@@ -0,0 +1,237 @@
+## Vulnerable Application
+
+This module requests TGT/TGS tickets from the KDC. Two ACTIONS are available:
+
+- **TGT**: legally request a TGT from the KDC given a password, a NT hash or
+  an encryption key. The resulting TGT will be stored in the loot.
+- **TGS**: legally request a TGS from the KDC given a password, a NT hash, an
+  encryption key or a cached TGT. If the TGT is not provided, it will request
+  it the same way the "TGT action" does. The resulting TGT and the TGS will be
+  stored in the loot.
```

Comment:
I'm pretty confident that before this work is moved into the `branch` that the tickets won't be stored in loot any more. Can we omit that detail while we're looking at it so we don't forget about it later.